tests/DocumentationExamplesTest.php

```diff
@@ -1180,6 +1184,223 @@ public function testIndex_example_2()
         $this->assertEquals('cuisine_1_name_1', $indexName);
     }
 
+    // Start Transactions Intro Example 1
+    private function updateEmployeeInfo1(Manager $manager, Session $session)
+    {
+        $employees = (new Database($manager, "hr"))->employees;
+        $events = (new Database($manager, "reporting"))->events;
+
+        $session->startTransaction([
+            'readConcern' => new ReadConcern("snapshot"),
+            'writeConcern' => new WriteConcern(WriteConcern::MAJORITY)
+        ]);
+
+        try {
+            $employees->updateOne(
+                ['employee' => 3],
+                ['$set' => ['status' => 'Inactive']],
+                ['session' => $session]
+            );
+            $events->insertOne(
+                ['employee' => 3, 'status' => [ 'new' => 'Inactive', 'old' => 'Active']],
+                ['session' => $session]
+            );
+        } catch (\MongoDB\Driver\Exception\Exception $error) {
+            echo "Caught exception during transaction, aborting.\n";
+            $session->abortTransaction();
+            throw $error;
+        }
+
+        while (true) {
+            try {
+                $session->commitTransaction();
+                echo "Transaction committed.\n";
+                break;
+            } catch (\MongoDB\Driver\Exception\CommandException $error) {
+                $resultDoc = $error->getResultDocument();
+
+                if (isset($resultDoc->errorLabels) && in_array('UnknownTransactionCommitResult', $resultDoc->errorLabels)) {
+                    echo "UnknownTransactionCommitResult, retrying commit operation ...\n";
+                    continue;
+                } else {
+                    echo "Error during commit ...\n";
+                    throw $error;
+                }
+            } catch (\MongoDB\Driver\Exception\Exception $error) {
+                echo "Error during commit ...\n";
+                throw $error;
+            }
+        }
+    }
+    // End Transactions Intro Example 1
+
+    public function testTransactions_intro_example_1()
+    {
```

Comment:
Let's add skips to all three test methods that utilize transactions:

```
if ($this->getPrimaryServer()->getType() === Server::TYPE_STANDALONE) {
    $this->markTestSkipped('Transactions are not supported on standalone servers');
}

if (version_compare($this->getFeatureCompatibilityVersion(), '4.0', '<')) {
    $this->markTestSkipped('Transactions are only supported on FCV 4.0 or higher');
}
```

Reply:
Done